blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -248,6 +248,7 @@ servererror.errors.message=Message: {0}
 servererror.message=Message: {0}
 servererror.title=An unknown error has occurred. You may try again.
 Show.complete.logs=Show complete log
-inputStep.error.message=This pipeline uses input types that are unsupported. \nUse [Jenkins Classic]({0}) to resolve this input step.
+inputStep.error.linktext=Resolve Input
```

Comment:
That will lead to `inputStep.error.linktext` being `Resolve Input` in all language other then `ru`